docs/funcx_upgrade.rst

```diff
@@ -2,46 +2,240 @@
+
+.. note::
+  An alternate upgrade path is to install the latest ``funcx / funcx-endpoint``
+  packages which are a wrapper around ``globus-compute-sdk`` and
+  ``globus-compute-endpoint`` respectively. This should work as is for most
+  users. However the funcx* packages are meant to be a temporary convenience
+  and are soon to be deprecated.
 
-The Globus team is renaming funcX to Globus Compute in order centralize our
-infrastructure under a single umbrealla.
 
 funcX SDK
 ^^^^^^^^^
 
-The `funcx` PyPI package was formerly the funcX SDK.  This is now named the `Globus
-Compute SDK` and is `available on PyPI <https://pypi.org/project/globus-compute-sdk/>`_
-under the same name.
+The ``funcx`` PyPI package was formerly the funcX SDK.  The SDK has been renamed
+to the ``Globus Compute SDK`` and is `available on PyPI <https://pypi.org/project/globus-compute-sdk/>`_.
 
-If you currently have funcX installed, we recommend these steps to upgrade to
-Globus Compute:
+If you currently have ``funcx`` installed, we recommend these steps for upgrading:
 
   | $ pip uninstall funcx
   | $ mv ~/.funcx ~/.globus_compute   # Optional
-  | $ Install Globus Compute SDK in its own venv `as detailed here <quickstart.rst#_install_gc_sdk>`__
+  | $ `Install Globus Compute SDK in its own venv <quickstart.html#installing-the-globus-compute-endpoint-optional>`_
+
+SDK Usage Changes
+-----------------
+
+Most users utilize only the ``FuncXClient`` and ``FuncXExecutor`` classes.
+These have been renamed to ``Client`` and ``Executor`` respectively. The
+required import changes are:
+
+.. code-block:: python
+
+  from funcx import FuncXClient
+  from funcx import FuncXExecutor
 
-The `funcx` package is still available on PyPI but will merely be a wrapper
-around the Globus Compute SDK.  The wrapper will only be available for
-a limited time to assist users as an easier migration path.
+to
+
+.. code-block:: python
+
+  from globus_compute_sdk import Client
+  from globus_compute_sdk import Executor
+
+All other module and classes that had ``"FuncX"`` in their names have been renamed
+to ``Compute*``.  These changes `are detailed below <#SDK-Module-and-Class-Changes>`_.
+
+The directory used for local cache storage is now ``~/.globus_compute`` instead of
+``~/.funcx``.  We recommend renaming ~/.funcx to ``~/.globus_compute`` after
+installing the new package.  Otherwise, first usage of the new SDK will
+automatically rename it and create a symlink from the old to the new.
+
+For other, less relevant changes, `also see below <#SDK-Module-and-Class-Changes>`_.
 
 funcX Endpoint
 ^^^^^^^^^^^^^^
 
-`funcx-endpoint` on PyPI was the former funcX endpoint package.  This is now called
-the `Globus Compute Endpoint` and is
-`available on PyPI <https://pypi.org/project/globus-compute-client/>`_.
+`funcx-endpoint` on PyPI was the former funcX endpoint package.  This is now
+`Globus Compute Endpoint` and is available on
```

Comment:
Should these be double-backticked?
```diff
- `funcx-endpoint`
+ ``funcx-endpoint``
```
And similar for `Globus Compute Endpoint`?